mitxpro/settings.py

```diff
@@ -1228,12 +1197,57 @@
 )
 
 # Hubspot sync settings
-HUBSPOT_API_KEY = get_string(
-    name="HUBSPOT_API_KEY", default=None, description="API key for Hubspot"
+MITOL_HUBSPOT_API_PRIVATE_TOKEN = get_string(
+    name="MITOL_HUBSPOT_API_PRIVATE_TOKEN",
+    default=None,
+    description="Hubspot private token to authenticate with API",
+)
+MITOL_HUBSPOT_API_RETRIES = get_int(
+    name="MITOL_HUBSPOT_API_RETRIES",
+    default=3,
+    description="Number of times to retry a failed hubspot API request",
+)
+MITOL_HUBSPOT_API_ID_PREFIX = get_string(
+    name="MITOL_HUBSPOT_API_ID_PREFIX",
+    default="XPRO",
+    description="The prefix to use for hubspot unique_app_id field values",
 )
-HUBSPOT_ID_PREFIX = get_string(
-    name="HUBSPOT_ID_PREFIX", default="xpronew", description="Hub spot id prefix."
+HUBSPOT_PIPELINE_ID = get_string(
+    name="HUBSPOT_PIPELINE_ID",
+    default="default",
+    description="Hubspot ID for the ecommerce pipeline",
 )
+HUBSPOT_MAX_CONCURRENT_TASKS = get_int(
+    name="HUBSPOT_MAX_CONCURRENT_TASKS",
+    default=5,
+    description="Max number of concurrent Hubspot tasks to run",
+)
+HUBSPOT_TASK_DELAY = get_int(
+    name="HUBSPOT_TASK_DELAY",
+    default=1000,
+    description="Number of milliseconds to wait between consecutive Hubspot calls",
+)
+HUBSPOT_CONFIG = {
+    "HUBSPOT_NEW_COURSES_FORM_GUID": get_string(
+        name="HUBSPOT_NEW_COURSES_FORM_GUID",
+        default="b9220dc1-4e48-4097-8539-9f2907f18b1e",
```

Comment:
I'm not sure if this and the other hubspot keys below should have defaults? That could cause something to get missed if we lose configuration in production since it won't error.

Reply:
These were just moved from one section of the `settings.py` file to another (so as to lump all hubspot settings together), but I agree that maybe these should probably just be in salt.  Maybe in a separate PR or would you rather have it done now?

Reply:
https://github.com/mitodl/mitxpro/issues/2453